api/src/middlewares/services/StaffService/StaffService.ts

```diff
@@ -81,21 +81,23 @@ export class StaffService {
 
   getEmailBody(data: { fields: FormField[]; payment?: PaymentViewModel; reference: string }, template: SESEmailTemplate, type: FormType) {
     const { fields, payment, reference } = data;
-    const remapped = remappers.affirmation(fields);
+    const remapped = remappers[type](fields);
 
     const { information } = remapped;
 
-    const reordered = reorderers.affirmation(remapped);
+    const reordered = reorderers[type](remapped);
     const country = getAnswerOrThrow(information, "country");
     const post = information.post?.answer;
+    const oathType = type === "affirmation" ? getAnswerOrThrow(information, "oathType") : undefined;
+    const jurats = type === "affirmation" ? getAnswerOrThrow(information, "jurats") : undefined;
```

Comment:
I won't block the PR on this, but 

assigning undefined to a variable is redundant, and ternaries get more difficult to read as the statements get longer. 

```suggestion
    let oathtype;
    let jurats;
    
    if(type === "affirmation") {
      oathType = getAnswerOrThrow(information, "oathType")
      jurats = getAnswerOrThrow(information, "jurats");
    }
    
```